src/main/java/online/talkandtravel/repository/ChatRepository.java

```diff
@@ -24,6 +25,9 @@ public interface ChatRepository extends JpaRepository<Chat, Long> {
 
   Page<Chat> findAllByChatType(ChatType chatType, Pageable pageable);
 
+  @Query("SELECT COUNT(*) FROM Message m WHERE m.id > :lastReadMessageId AND m.chat.id = :chatId")
```

Comment:
we should discuss this in telegram. This is a very questionable decision.
What if newer messages will have lower number of Id? I mean was 101, 102 but next apper 56. This method will fail. Do we really need @Query here?

Reply:
Newer messages always have a greater number in any case